include/powerloader/download_target.hpp

```diff
@@ -240,8 +240,8 @@ namespace powerloader
         std::size_t m_byterange_start = 0;
         std::size_t m_byterange_end = 0;
         std::string m_range;
-        std::ptrdiff_t m_expected_size = 0;
-        std::ptrdiff_t m_orig_size = 0;
+        std::uintmax_t m_expected_size = 0;
+        std::uintmax_t m_orig_size = 0;
```

Comment:
What was the warning that lead you to replace `ptrdiff_t` by `uintmax_t`? `ptrdiff_t` can have bigger values than `unitmax_t` on 64bit platforms so that's not a trivial change.

Reply:
It's what fs::filesize returns